### PR TITLE
Add @stricjs/kit template

### PR DESCRIPTION
**Pull request by aquapi**

### CODE REVIEW

File: stric/package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@bun-examples/stric",
+  "module": "index.ts",
+  "devDependencies": {
+    "bun-types": "^0.5.0",
+    "@types/react": "^18.0.28"
+  },
+  "type": "module",
+  "dependencies": {
+    "@stricjs/jsx": "^1.0.4",
```
Comment (SaltyAom) [inline stric/package.json:10]: This is a reminder, but you may want to use `latest` instead a specific version, so when the new version released, you don't have to create a PR to edit the version.

File: stric/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+This is the project bootstrapped using `@stricjs/kit`.
```
Comment (SaltyAom) [inline stric/README.md:1]: If possible, please use the following template to match the convention, I think you can add some additional detail after the template, can you confirm? @Kapsonfire-DE 

> # Stricjs with Bun runtime
> 
> ## Getting Started
> To get started with this template, simply paste this command into your terminal:
> ```bash
> bun create strictjs./strictjs-example
> ```
>
> ## Development
> To start the development server run:
> ```bash
> bun run dev
>```
> 
> Open http://localhost:3000/ with your browser to see the result.

**Review by SaltyAom — CHANGES_REQUESTED** [review]

File: stric/README.md (hunk 2 above)
Comment (xHyroM) [reply stric/README.md:1]: that would be fine 👍

Comment (aquapi) [reply stric/README.md]: Added :)

File: stric/package.json (hunk 1 above)
Comment (aquapi) [reply stric/package.json]: Added :)

File: stric-react/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@bun-examples/stric-react",
+  "module": "index.ts",
+  "devDependencies": {
+    "bun-types": "^0.5.0",
```
Comment (xHyroM) [inline stric-react/package.json:5]: maybe i would use `latest` here as well

bun is at a stage where everyone is upgrading right away, not like node where you have people using v16, v18, v19

**Pull request merged**